ndindex/__init__.py

```diff
@@ -24,6 +24,10 @@
 
 __all__ += ['IntegerArray']
 
+from .booleanarray import BooleanArray
+
+__all__ += ['BooleanArray']
```

Comment:
It is odd to build up `__all__` programatically like this.  Seems like there should be a single definition of it

Reply:
I think it's a common way to do it. It keeps all the `__all__` items next to their imports.

Reply:
Hmm weird... I don't think I have ever seen it done this way